client/web/src/enterprise/batches/create/CreateOrEditBatchChangePage.tsx

```diff
@@ -143,72 +153,103 @@ const CreatePage: React.FunctionComponent<CreatePageProps> = ({ namespaceID, set
+                    className="flex-1 pb-2"
+                    description="Run custom code over hundreds of repositories and manage the resulting changesets."
+                    annotation={
+                        <FeedbackBadge status="experimental" feedback={{ mailto: 'support@sourcegraph.com' }} />
+                    }
+                />
+                <Form className="my-4 pb-5" onSubmit={handleCreate}>
+                    <Container className="mb-4">
+                        {error && <ErrorAlert error={error} />}
+                        <NamespaceSelector
+                            namespaces={namespaces}
+                            selectedNamespace={selectedNamespace.id}
+                            onSelect={setSelectedNamespace}
+                        />
+                        <Input
+                            label="Batch change name"
+                            value={nameInput}
+                            onChange={onNameChange}
+                            pattern={String(NAME_PATTERN)}
+                            required={true}
+                            status={nameValid === undefined ? undefined : nameValid ? 'valid' : 'error'}
+                        />
+                        <small className="text-muted">
+                            Give it a short, descriptive name to reference the batch change on Sourcegraph. Do not
+                            include confidential information.{' '}
+                            <span className={classNames(nameValid === false && 'text-danger')}>
+                                Only regular characters, _ and - are allowed.
+                            </span>
+                        </small>
+                        <hr className="my-3" />
+                        <h3 className="text-muted">
+                            Visibility <InfoCircleOutlineIcon className="icon-inline" data-tooltip="Coming soon" />
+                        </h3>
+                        <div className="form-group text-muted mb-1">
+                            <RadioButton
+                                name="visibility"
+                                value="public"
+                                className="mr-2"
+                                checked={true}
+                                disabled={true}
+                                label="Public"
+                                aria-label="Public"
+                            />
+                        </div>
+                        <div className="form-group text-muted mb-0">
+                            <RadioButton
+                                name="visibility"
+                                value="private"
+                                className="mr-2 mb-0"
+                                disabled={true}
+                                label={
+                                    <>
+                                        Private <LockIcon className="text-warning icon-inline" />
```

Comment:
Accessibility suggestion (I know I've been dropping this sometimes too 👀)

```suggestion
                                        Private <LockIcon className="text-warning icon-inline" aria-hidden={true} />
```

Reply:
Thanks for the reminder! I keep being too lax about this.